io.dbeaver.DBeaverCommunity.appdata.xml

```diff
@@ -30,6 +30,55 @@
         </screenshot>
     </screenshots>
     <releases>
+        <release version="7.1.2" date="2020-07-05">
+            <description>
+                <p>Data viewer:</p>
+                <ul>
+                    <li>Now we refresh entire row after cell value update (handy if you have triggers)</li>
+                    <li>Generate SQL context menu was fixed</li>
+                    <li>Horizontal scroll with mouse wheel support was added</li>
+                    <li>Export data in source code (PHP arrays) format was added</li>
+                    <li>Bug with Ctrl+Backspace handler was fixed</li>
+                </ul>
+                <p>SQL Editor:</p>
+                <ul>
+                    <li>Query cancel handler was improved (problem with UI stuck was fixed)</li>
+                    <li>Now we show query information in the result tab tooltip</li>
+                    <li>Query parser was fixed (active query extraction for Ctrl+Enter action)</li>
+                    <li>Template variables (${word_selection} and ${line_selection}) support was fixed</li>
+                </ul>
+                <p>Navigator:</p>
+                <ul>
+                    <li>Icons for "Create object" actions were fixed</li>
+                    <li>Main menu File items were reordered</li>
+                    <li>Heading and trailing spaces trim was added to all edit dialogs/panels</li>
+                </ul>
+                <p>Session manager: problem with multiple windows was fixed
+                SQL Server: Database tools (index rebuild, triggers disable) were redesigned
+                Firebird: new table and table column creation was fixed
+                PostgreSQL: Table partition DDL was fixed
+                MySQL: CHECK constraints edit support was added
+                Oracle:</p>
```

Comment:
In this case, 'Oracle' should be separate `<p>` tag.
Update script still needs some refining in order to be automated...

Reply:
Got it, I missed those parts.